mantidproject/mantid
# Port Indirect Bayes to the Workbench

~~Preferably merge after #26522~~
Only review on windows
-----------------------------------
**Description of work.**
This PR ports the Indirect Bayes interface to the workbench. 

**Release notes will follow in the PR which ports Data Analysis**

**To test:**
Workbench and MantidPlot:
1. `Interfaces`->`Indirect`->`Bayes`
2. Load the data below
3. Click `Run` and wait.
4. Click `Plot` and external plots should appear.

**Data Files**

**ResNorm**
[irs26173_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2491289/irs26173_graphite002_red.zip)
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2491290/irs26173_graphite002_res.zip)

**Quasi**
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2491291/irs26176_graphite002_red.zip)
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2491292/irs26173_graphite002_res.zip)

**Stretch**
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2491291/irs26176_graphite002_red.zip)
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2491292/irs26173_graphite002_res.zip)

Fixes #25984

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
